docs/docs/data-sources/graphql.md

```diff
@@ -23,19 +22,52 @@ The following optional parameters are also supported:
    | URL params   | Additional query string parameters|
    | headers      | Any headers the GraphQL source requires|
 
+<div style={{textAlign: 'center'}}>
 
+![ToolJet - Data source - GraphQl](/img/datasource-reference/graphql/graphql-ds.png)
 
-<img class="screenshot-full" src="/img/datasource-reference/graphql/add-source.gif" alt="ToolJet - GraphQL connection" height="420"/>
+</div>
 
-Click on the 'Save' button to save the data source.
+Click on the **Save** button to save the data source.
 
 ## Querying GraphQL
-Click on `+` button of the query manager at the bottom panel of the editor and select the GraphQL endpoint added in the previous step as the  data source. 
 
-<img class="screenshot-full" src="/img/datasource-reference/graphql-query.png" alt="ToolJet - GraphQL connection" height="420"/>
+Click on `+` button of the query manager at the bottom panel of the editor and select the GraphQL endpoint added in the previous step as the  data source.
+
+<div style={{textAlign: 'center'}}>
+
+![ToolJet - Data source - GraphQl](/img/datasource-reference/graphql/graphql-query.png)
+
+</div>
 
 Click on the 'run' button to run the query. NOTE: Query should be saved before running.
 
 :::tip
 Query results can be transformed using transformations. Read our transformations documentation to see how: [link](/docs/tutorial/transformations)
-:::
\ No newline at end of file
+:::
+
+## Including variables in GraphQL queries
+
+You can inject parameters directly into the query using Javascript. 
```

Comment:
we have added `variables` field, this is no more required.

Reply:
Awesome! 
@Navaneeth-pk I think we can now close this PR and the issue

Reply:
Thanks! removed these changes.